### PR TITLE
Add Sister Outsider Study Guide

### PR DESCRIPTION
**Pull request by lizgene**
- Add new study guide for sister outsider by audre lorde
- Add categories and update meta data for #blacklivesmatter study guide
- Address $base-spacing issue discussed in https://github.com/radicalbarracuda/sf-feminist-book-club/pull/10

bitmoji

### CODE REVIEW

**Comment by lizgene** [comment]
@pamo Re: Your question on quotes - I prefer dumb `"quotes"` so am happy to stick to that and standardize whenever new study guides are brought in from other sources. 👍

**Comment by pamo** [comment]
Cool, I'll wait for you to find and replace 👍🏽

**Comment by lizgene** [comment]
@pamo Have found and have replaced 👍

File: _posts/2017-03-19-audre-lorde-sister-outsider-study-guide.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+and learning from the genuine anger of others without guilt or
+defensiveness.
+
+"Anger expressed and translated into action in the service of our vision
+and our future is a liberating and strengthening act of clarification,
+for it is in the painful process of this translation that we identify
+who are our allies with whom we have grave differences, and who are our
+genuine enemies."
+
+"…guilt is just another name for impotence, for defensiveness
```
Comment (pamo) [inline _posts/2017-03-19-audre-lorde-sister-outsider-study-guide.md]: Should we change these ellipsis to be three separate periods?

**Review by pamo — CHANGES_REQUESTED** [review]

File: _posts/2017-03-19-audre-lorde-sister-outsider-study-guide.md (hunk 1 above)
Comment (lizgene) [reply _posts/2017-03-19-audre-lorde-sister-outsider-study-guide.md]: I totally wouldn't have noticed this. Nice catch 👍

**Review by pamo — APPROVED** [review]

**Pull request merged**